tests/schemas/test_coreapi.py

```diff
@@ -1372,3 +1380,46 @@ def test_schema_handles_exception():
     response.render()
     assert response.status_code == 403
     assert b"You do not have permission to perform this action." in response.content
+
+
+@pytest.mark.skipif(not coreapi, reason='coreapi is not installed')
+def test_coreapi_deprecation():
+    with pytest.warns(RemovedInDRF315Warning):
```

Comment:
Could look to assert the warning message (using `match`?), along the lines of `assertRaisesMessage`. However, given this is the only deprecation with a 315 warning I'm not sure we need to be that accurate?  

https://docs.pytest.org/en/stable/warnings.html#warns